website/docs/r/shared_image.html.markdown

```diff
@@ -74,6 +74,8 @@ The following arguments are supported:
 
 * `release_note_uri` - (Optional) The URI containing the Release Notes associated with this Shared Image.
 
+* `os_state` - (Optional) The state of Operating System present in this Shared Image. Possible values are `Generalized` and `Specialized`. Defaults to `Generalized`. Changing this forces a new resource to be created.
```

Comment:
After some internal discussion could we rename this to `generalized = bool` ? this is at heart a boolean true/false and os_state really doesn't convey what its doing.

Reply:
Great idea! I will take care of this.